src/Faker/Generator.php

```diff
@@ -365,6 +362,36 @@ public function isbn13(): string
         return $this->ext(Extension\BarcodeExtension::class)->isbn13();
     }
 
+    public function numberBetween($int1 = 0, $int2 = 2147483647): int
+    {
+        return $this->ext(Extension\NumberExtension::class)->numberBetween($int1, $int2);
+    }
+
+    public function randomDigit(): int
+    {
+        return $this->ext(Extension\NumberExtension::class)->randomDigit();
+    }
+
+    public function randomDigitNot($except): int
+    {
+        return $this->ext(Extension\NumberExtension::class)->randomDigitNot($except);
+    }
+
+    public function randomDigitNotNull(): int
+    {
+        return $this->ext(Extension\NumberExtension::class)->randomDigitNotNull();
+    }
+
+    public function randomFloat($nbMaxDecimals = null, $min = 0, $max = null): float
```

Comment:
Can we add a DocBlock here?

Reply:
I've added dockblocks to the generator ext methods now (will do so for the other extension methods in a separate PR).